en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -472,4 +473,75 @@ To create a For Each Loop:
+
+To watch a step: 
+
+. <<access-error-handling,Access the Error Handler in your flow>>.
+.  In the *Error Handler* card, in the *Watch* section, click the plus sign and select the action in your flow that you want to watch or drag the step that you want to watch into the *Watch* section. +
+The selected action appears in the *Watch* section. If an error occurs, the flow moves to the actions selected in the *On Error* section.
+. To add additional actions to watch, repeat the previous step.
+
+=== Configure Composer to Handle Errors 
+
+You can configure Composer to handle errors encountered in the *Watch* section. If an error occurs in a watched action, the *On Error* section begins processing the flow based on your configuration.  You can add one or more steps to run in the *On Error* section. 
+
+==== Before You Begin 
+
+* By default, the *On Error* section allows for one condition to be met; however, you can add as many conditions as you want in this section. 
+* When adding a new condition, the *On Error* condition field value is set to `None` by default.  Leaving this value set to `None` returns any errors that may occur in the *Watch* section and processes all steps inside this *On Error* section. 
+* When a condition is met in the *On Error* section, the Error Handler processes the conditions in sequential order and stops once the first condition met. After the first condition is met, any subsequent steps are skipped. If you want the process to continue, you can add an additional *On Error* section branch inside your existing *On Error* section.
+* When xref:ms_composer_test_flow.adoc[testing your flow], any errors caught by the Error Handler appear in the right pane of the action card. Successful steps are marked with a green check and failed steps are marked with a red X. 
+* When an error occurs in a watched action, the error information is saved as a data pill that can then be used for subsequent notification or record-updating actions. For example, if an error occurs, you can designate an action to send an email to your team containing the error information from the data pill.
+
+==== Steps 
+
+To add steps to the *On Error* section of your flow: 
+
+. <<watch-a-step,Select a step to watch>>.
+. In the *Error Handler* card, in the *On Error* section: 
+.. In the *Conditions to run this branch* drop-down, select one of the following conditions and then enter the requested criteria: 
+... `AND`: All conditions must be met.
+... `OR`: Any conditions can be met.
+... `None`: No conditions must be met and all errors are returned.
+... `Custom Logic`: <<use-custom-logic,Customize logic>> to designate how conditions will be met. 
+.. In the *After Running* drop-down, select: 
+... *Stop*: After processing all steps in the *On Error* section, errors are flagged and the flow stops. The *Status* field of the flow is marked as `FAILED` on the <<flow-details-page,*Run History*>> page.  
```

Comment:
Unordered list, I think.

Reply:
"...Composer flags errors and stops the flow"
"On the *Run History" page, the *Status* field of the flow shows `FAILED`."

Reply:
Fixed!